api/src/pcapi/utils/sentry.py

```diff
@@ -8,7 +11,15 @@
 from pcapi.utils.health_checker import read_version_from_file
 
 
-def init_sentry_sdk():  # type: ignore [no-untyped-def]
+def ignore_flask_shell_event(
+    event: dict[str, typing.Any], _hint: dict[str, typing.Any]
+) -> dict[str, typing.Any] | None:
+    if sys.argv and len(sys.argv) == 2 and "flask" in sys.argv[0] and sys.argv[1] == "shell":
```

Comment:
(détail) Je ne pense pas que `sys.argv` puisse être autre chose qu'une liste. On devrait donc pouvoir simplifier en

```suggestion
    if len(sys.argv) >= 2 and "flask" in sys.argv[0] and sys.argv[1] == "shell":
```

Au passage, je propose `>= 2` car, apparemment, on peut donner des arguments à `flask shell` : 

```
$ flask shell --help
Usage: flask shell [OPTIONS] [IPYTHON_ARGS]...
```